NorthstarDedicatedTest/logrotation.cpp

```diff
@@ -0,0 +1,80 @@
+#include "pch.h"
+#include <filesystem>
+#include <string>
+#include <time.h>
+#include <iostream>
+#include <regex>
+#include "configurables.h"
+
+namespace fs = std::filesystem;
+using namespace std;
```

Comment:
^ ...

Reply:
I think what pg meant is that these namespace shortening are helpful but they shouldn't really be introduced in the code base since it's a Good Practice ™️